src/pages/credit-order/Forms/CreditOrderForm.js

```diff
@@ -598,13 +590,21 @@ export default function CreditOrderForm({ labels, access, recordId, plantId, use
       name: 'amount',
       props: {
         readOnly: true,
-        mandatory: true,
-        disabled: !formik.values.corId || isClosed
+        mandatory: true
       },
       width: 130
     }
   ]
 
+  async function getCurrencyGoc(currencyId) {
+    const res = await getRequest({
+      extension: RemittanceSettingsRepository.CorrespondentCurrency.get,
+      parameters: `_corId=${formik.values.corId}&_currencyId=${currencyId}`
+    })
+
+    return res?.record?.goc
```

Comment:
return res?.record and then get the goc value in the update function. if the res.record is null then you can consider that goc is false